facedancer/devices/ftdi.py

```diff
@@ -210,6 +224,24 @@ def handle_get_latency_timer_request(self, request):
         # Per Travis Goodspeed, this is a "bullshit value".
         request.reply(b'\x01')
 
+    @vendor_request_handler(number=144)
+    def handle_control_request_144(self, request):
```

Comment:
Since we know what the request is, we can use a better name for the method:

```suggestion
    def handle_read_eeprom_request(self, request):
```